build-tools/scripts/XAVersionInfo.targets

```diff
@@ -5,7 +5,52 @@
   <UsingTask AssemblyFile="$(MSBuildThisFileDirectory)..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"  TaskName="Xamarin.Android.BuildTools.PrepTasks.GitCommitHash" />
   <UsingTask AssemblyFile="$(MSBuildThisFileDirectory)..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"  TaskName="Xamarin.Android.BuildTools.PrepTasks.GitCommitsInRange" />
   <UsingTask AssemblyFile="$(MSBuildThisFileDirectory)..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"  TaskName="Xamarin.Android.BuildTools.PrepTasks.GitCommitTime" />
-  <Target Name="GetXAVersionInfo">
+  <UsingTask AssemblyFile="$(MSBuildThisFileDirectory)..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"  TaskName="Xamarin.Android.BuildTools.PrepTasks.GitCommitInfo" />
+  <Target Name="_GetSubmodulesVersionInfo">
+    <ItemGroup>
+      <!-- If anything changes in this list, make sure to update:
+	   src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets (the _GenerateXACommonProps target)
+	   src/Xamarin.Android.Build.Tasks/Xamarin.Android.BuildInfo.props.in
+	   build-tools/create-vsix/create-vsix.targets (the _CreateDependencies target)
+      -->
+      <_SubmoduleBranchInfo Include="external/mono">
+	<OutputPropertyName>_BuildInfo_MonoCommit</OutputPropertyName>
```

Comment:
XML formatting: this should be indented another 2 spaces, as well as all other `<OutputPropertyName/>` values.